pkg/bridge/ai/internal/oai/model.go

```diff
@@ -1,6 +1,14 @@
-package openai
+package oai
 
-import "github.com/yomorun/yomo/ai"
+import (
+	"github.com/yomorun/yomo/ai"
+	"github.com/yomorun/yomo/core/metadata"
+)
+
+// ILLMClient is the interface for OpenAI API client
+type ILLMClient interface {
```

Comment:
It is better to name as `LLMClient`, see https://go.dev/doc/effective_go#interface-names， avoid `I` prefix.

Reply:
> It is better to name as `LLMClient`, see [https://go.dev/doc/effective_go#interface-names，](https://go.dev/doc/effective_go#interface-names%EF%BC%8C) avoid `I` prefix.

changed to `OpenAIRequester`